android-sdk/Dockerfile

```diff
@@ -54,7 +54,7 @@ RUN mkdir -p ${ANDROID_SDK_ROOT}/cmdline-tools && \
 ENV JAVA_HOME /usr/lib/jvm/java-${JDK_VERSION}-openjdk-amd64
 ENV GRADLE_HOME /opt/gradle
 ENV KOTLIN_HOME /opt/kotlinc
-ENV PATH ${PATH}:${GRADLE_HOME}/bin:${KOTLIN_HOME}/bin:${ANDROID_SDK_ROOT}/cmdline-tools/tools/bin:${ANDROID_SDK_ROOT}/platform-tools:${ANDROID_SDK_ROOT}/emulator
+ENV PATH ${PATH}:${GRADLE_HOME}/bin:${KOTLIN_HOME}/bin:${ANDROID_SDK_ROOT}/cmdline-tools/cmdline-tools/bin:${ANDROID_SDK_ROOT}/platform-tools:${ANDROID_SDK_ROOT}/emulator
```

Comment:
Same here, we can remove the redundant `cmdline-tools`.

* Line 48: mkdir of `${ANDROID_SDK_ROOT}` without `cmdline-tools`
* Line 50: unzip the SDK package to `${ANDROID_SDK_ROOT}`, instead of its sub `cmdline-tools` directory
* Line 57: `${ANDROID_SDK_ROOT}/cmdline-tools/cmdline-tools/bin` -> `${ANDROID_SDK_ROOT}/cmdline-tools/bin`